From 1845b514511398cc0ba219a47958f078a2f7f67d Mon Sep 17 00:00:00 2001
From: Shiji Yang <yangshiji66@outlook.com>
Date: Tue, 8 Jul 2025 19:50:53 +0800
Subject: [PATCH] arm: mediatek: add back mt7622 LK image header

The LK image header was lost since commit df3ab898f644
("arm: mediatek: merge board Kconfigs into mach-mediatek").

Signed-off-by: Shiji Yang <yangshiji66@outlook.com>
---
 arch/arm/mach-mediatek/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/mach-mediatek/Kconfig
+++ b/arch/arm/mach-mediatek/Kconfig
@@ -158,7 +158,7 @@ config MTK_BROM_HEADER_INFO
 	string
 	default "media=nor" if TARGET_MT8518 || TARGET_MT8512 || TARGET_MT7629
 	default "media=emmc" if TARGET_MT8516 || TARGET_MT8365 || TARGET_MT8183
-	default "lk=1" if TARGET_MT7623
+	default "lk=1" if TARGET_MT7622 || TARGET_MT7623
 
 config MTK_TZ_MOVABLE
 	select ARCH_MISC_INIT
